Source/Core/DolphinQt2/ToolBar.cpp

```diff
@@ -93,13 +94,15 @@ void ToolBar::UpdateIcons()
 {
   QString dir = Settings().GetThemeDir();
 
-  m_open_action->setIcon(QIcon(QStringLiteral("open.png").prepend(dir)));
-  m_paths_action->setIcon(QIcon(QStringLiteral("browse.png").prepend(dir)));
-  m_play_action->setIcon(QIcon(QStringLiteral("play.png").prepend(dir)));
-  m_pause_action->setIcon(QIcon(QStringLiteral("pause.png").prepend(dir)));
-  m_stop_action->setIcon(QIcon(QStringLiteral("stop.png").prepend(dir)));
-  m_fullscreen_action->setIcon(QIcon(QStringLiteral("fullscreen.png").prepend(dir)));
-  m_screenshot_action->setIcon(QIcon(QStringLiteral("screenshot.png").prepend(dir)));
-  m_config_action->setIcon(QIcon(QStringLiteral("config.png").prepend(dir)));
-  m_controllers_action->setIcon(QIcon(QStringLiteral("classic.png").prepend(dir)));
+  const QString suffix = DPI::GetScaledIconSuffix(m_scale, 2);
+
+  m_open_action->setIcon(QIcon(QStringLiteral("open%1.png").arg(suffix).prepend(dir)));
```

Comment:
I think there's a way to tell QIcon about those. I recall something about adding files to it manually, with explicit devicepixelratios?